ecoscope/plotting/plot.py

```diff
@@ -268,3 +268,41 @@ def plot_seasonal_dist(ndvi_vals, cuts, bandwidth=0.05, output_file=None):
     if output_file:
         fig.write_image(output_file)
     return fig
+
+
+def stacked_bar_chart(data, layout_kwargs=None):
+    """
+    Creates a stacked bar chart from the provided EcoPlotData object
+    Parameters
+    ----------
+    data: ecoscope.Plotting.EcoPlotData
+        The data to plot, counts categorical data.y_col values for data.x_col
+    layout_kwargs: dict
+        Additional kwargs passed to plotly.go.Figure(layout)
+    Returns
+    -------
+    fig : plotly.graph_objects.Figure
+        The plotly bar chart
+    """
+    # TODO cleanup EPD defaults
+    data.style.pop("mode")
+
+    fig = go.Figure(layout=layout_kwargs)
+
+    x_axis_name = data.x_col
+    y_axis_name = data.y_col
+
+    agg = data.grouped[y_axis_name].count().to_frame("count").unstack(fill_value=0).stack().reset_index()
```

Comment:
Does it mean we can only draw stacked bar chart on `count`? How about `sum`?

Reply:
The goal was to suit the patrols dashboard specifically, which will need only `count`. That said I'll take a look at the effort to make this more generic

Reply:
I've updated the function with a string param `agg_function` which allows any `pandas.agg()` to be called